images/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG BASE_IMAGE=ubuntu:24.04
+ARG VARIANT=core
+ARG MODEL=generic
+ARG TRUSTED_BOOT=false
+
+
+FROM quay.io/kairos/kairos-init:v0.1.1 AS kairos-init
+FROM quay.io/luet/base:0.35.5 AS luet
+
+FROM ${BASE_IMAGE} AS base
+ARG VARIANT
+ARG MODEL
+ARG TRUSTED_BOOT
+COPY --from=kairos-init /kairos-init /kairos-init
+RUN /kairos-init -l debug -s install -m "${MODEL}" -v "${VARIANT}" -t "${TRUSTED_BOOT}"
+RUN /kairos-init -l debug -s init -m "${MODEL}" -v "${VARIANT}" -t "${TRUSTED_BOOT}"
+#RUN /kairos-init -l debug --validate -m "${MODEL}" -v "${VARIANT}" -t "${TRUSTED_BOOT}"
+RUN rm /kairos-init
+
+
+FROM base AS core
+
+FROM base AS standard
+COPY --from=luet /usr/bin/luet /usr/bin/luet
+RUN luet install -y k8s/k3s-$(grep -q alpine /etc/os-release && echo "openrc" || echo "systemd")
```

Comment:
can we have here a variable to build also k0s? ideally something like `KUBERNETES_DISTRO` or such?

Reply:
also, isn't it missing @${SOFTWARE_LUET_VERSION:-$SOFTWARE_VERSION} ? or am I missing something

Reply:
yes, it was discussed and currently we will NOT provide the way of building k3s/k0s bundled in the init, it has to be done by the user when building the image. This is easy as we have the frameowork so its just a simple luet install VERSION away.

I guess we can fix this down the line if we want to provide k3s/k0s support directly on the init part?